test/test-applications/regression/Reproduction.Wpf.ExpenseIt/ExpenseIt/ExpenseItDemo/ExpenseItDemo.csproj

```diff
@@ -128,16 +128,6 @@
   <ItemGroup>
     <None Include="App.config" />
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\..\..\..\..\src\Datadog.Trace.ClrProfiler.Managed.Core\Datadog.Trace.ClrProfiler.Managed.Core.csproj">
-      <Project>{d95d5e26-f32a-481d-a15a-ef7b3b56d2e0}</Project>
-      <Name>Datadog.Trace.ClrProfiler.Managed.Core</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\..\..\..\..\src\Datadog.Trace.ClrProfiler.Managed\Datadog.Trace.ClrProfiler.Managed.csproj">
-      <Project>{85f35aaf-d102-4960-8b41-3bd9cbd0e77f}</Project>
-      <Name>Datadog.Trace.ClrProfiler.Managed</Name>
-    </ProjectReference>
```

Comment:
This is great, thanks. I am also removing them in by branch, `lpimentel/merge-projects`, because all the code is merged into `Datadog.Trace.csproj`. I'm guessing we need to talk about how this PR affects my branch and what changes I should make there to keep code coverage working.